packages/appium/lib/types.d.ts

```diff
@@ -0,0 +1,27 @@
+import {AppiumConfiguration} from './appium-config';
+
+/*
+ * Utility types
+ */
+
+export type CamelCase<S extends string> =
```

Comment:
whoa did not know you could define restricted types of string based on templates!

Reply:
yeah, it's pretty new.  I don't really understand it too well; I asked a colleague for some assistance.